### DISCUSSION TITLE
BTLineWidth change location

### DISCUSSION THREAD

**Discussion by ProfStack** [artisan-roaster-scope/artisan#1096] [Q&A]
Wanting to have a thicker BT linewidth, but I can't find where to change it.  

I know where it is located in the SETUP file and can change it there, but there must be a location in 2.8.2 (Windows) to make the change.

I have looked many times and haven't found it.  Please help a "blind" man see.  Thanks.  :)

**Comment by MAKOMO** [comment]
Click here

<img width="687" alt="Screenshot 2023-02-07 at 20 17 28" src="https://user-images.githubusercontent.com/13909011/217344162-e15bff2a-27a7-4b49-94fe-9da08889c832.png">
<img width="722" alt="Screenshot 2023-02-07 at 20 19 37" src="https://user-images.githubusercontent.com/13909011/217344168-0d46de03-353e-4707-a9a9-8e36b74080eb.png">

**Reply by ProfStack** [reply]
Thank you, Marko.  The Windows page of this option is more compressed than the Apple version shown above.  But now I know where it is.  Much appreciated.
